src/components/Settings/GameBoardSettings.vue

```diff
@@ -27,6 +29,55 @@
       </select>
     </div>
 
+    <div class="setting">
+      <i class="fas fa-bars" />
+      <h5>Header position</h5>
+
+      <select v-model="value[platform.code].position" @change="$emit('save')">
+        <option
+          v-for="{ id, name } in positions"
+          :key="id"
+          :value="id"
+        >
+          {{ name }}
+        </option>
+      </select>
+    </div>
+
+    <div class="setting">
+      <i class="fas fa-shapes" />
+      <h5>Border Radius</h5>
+
+      <toggle-switch
+        id="borderRadius"
+        @change="$emit('save')"
+        v-model="value[platform.code].borderRadius"
+      />
+    </div>
+
+    <div class="setting">
+      <i class="fas fa-stream" />
+      <h5>Hide list backgrounds</h5>
+
+      <toggle-switch
+        id="hideListBackgrounds"
+        @change="$emit('save')"
+        v-model="value[platform.code].hideListBackgrounds"
+      />
+    </div>
```

Comment:
can we achieve this via themes? maybe setting the list background to be the same as the game board background?